core/src/primitives/rectangle/points.rs

```diff
@@ -1,4 +1,6 @@
-use crate::{geometry::Point, primitives::rectangle::Rectangle};
+//! Iterator over all points inside the rectangle.
+
```

Comment:
IMO we don't need to add doc comments to private modules, because they will never be seen in to docs.
```suggestion
```